ui/components/search-filter-view.jsx

```diff
@@ -1,26 +1,58 @@
 import React from 'react';
+import {Collapse} from 'react-collapse';
 
-export default function SearchFilterView(
-  { filterControls, pageContent, selectedFilters, tabs }) {
-  return (
-    <div className="clearfix">
-      <div className="sidebar col col-2 p2 no-print">
-        Search and filter
-        {filterControls}
-      </div>
-      <div className="main col col-10 pl4 border-left max-width-3">
-        <div className="tab-container no-print">
-          <span className="mr4">View:</span>
-          <ul className="organize-tabs list-reset inline-block">
-            {tabs}
-          </ul>
+export default class SearchFilterView extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {isOpened: false};
```

Comment:
Starting closed will mean non-JS users with small screens can't use these filters. It's not worth holding back progress here, but we may want to start open and close it within `componentDidMount()`, or use the `ConditionalRender` helper to handle that for us:
```jsx
<ConditionRender>
    <MobileFilters startOpen={true} />
    <MobileFilters startOpen={false} />
</ConditionRender>
```